eclair-core/src/main/scala/fr/acinq/eclair/payment/PaymentRequest.scala

```diff
@@ -67,6 +66,11 @@ case class PaymentRequest(prefix: String, amount: Option[MilliSatoshi], timestam
     case PaymentRequest.DescriptionHash(h) => Right(h)
   }.get
 
+  /**
+   * @return the payment metadata
```

Comment:
I expanded it a bit in https://github.com/ACINQ/eclair/pull/2063/commits/96ce001b35235e41cf80bb896e7850fc87df3125 to mention the feature, this way we can search for it in the codebase and find the explanation of how it can be used